pkgs/servers/monitoring/nagios/plugins/zfs.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, zfs
+, sudo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "check_zfs";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "zlacelle";
+    repo = "nagios_check_zfs_linux";
+    rev = version;
+    sha256 = "gPLCNt6hp4E94s9/PRgsnBN5XXQQ+s2MGcgRFeknXg4=";
+  };
+
+  buildInputs = [ python3 zfs sudo ];
+
+  installPhase = ''
+    runHook preInstall
+
+    substituteInPlace ${pname}.py \
```

Comment:
Remove this reference to `pname`. Use `check_zfs` directly.

Reply:
I can do that but what is the reason?

Reply:
First, it is ugly. Not a good technical reason, I know, but aesthetics is important.

Second, it is not always that the installed program name has the same name as the project name.
The most common example I can provide is a fork.
Indeed some people fork abandoned projects, changing their `pname` but keeping the file tree mostly intact. If by any reason we Nix packagers decide to change for such a "revamped fork", such ${pname} parameterization will be patched.

Third, usually we avoid unneeded and/or excessive parameterization (does this word exist?). 
`version` is parameterized because it usually composes useful and changing resources as links, tags, changelogs and alike, whereas `pname` does not have the same relationship with the code.